lib/connect.js

```diff
@@ -278,6 +278,10 @@ function connect(token, options) {
  * @property {boolean|CreateLocalTrackOptions} [audio=true] - Whether or not to
  *   get local audio with <code>getUserMedia</code> when <code>tracks</code>
  *   are not provided.
+ * @property {boolean} [enableAutomaticSubscription=true] - By default,
```

Comment:
@makarandp0 Can you replicate my changelog entry suggestion here? Also, `automaticSubscription`.